### PR TITLE
fix(Checkbox/Radio): Fixes regression from 5.1.1 update

### PR DESCRIPTION
**Pull request by mathewmorris**
Coming from feedback in https://github.com/heydoctor/web-app/pull/408

This will solve the regressions from that update while still keeping the features requested.

### CODE REVIEW

**Comment by mathewmorris** [comment]
@heydoctor/engineering Could I get a couple eyes on this when you get the chance?

File: src/Form/Checkbox.js [hunk 1]
```diff
@@ -111,47 +115,80 @@ const CheckboxLabel = createComponent({
...
-      margin-left: ${horizontal ? 12 : 0}px;
-      margin-top: ${horizontal ? 0 : 4}px;
-    }
+    return css`
+    display: inline-flex;
+    align-items: flex-start;
+    justify-content: flex-start;
+    margin-bottom: 4px;
+    user-select: none;
+    cursor: pointer;
```
Comment (erikshestopal) [inline src/Form/Checkbox.js]: stylistic nit: can we indent these styles here and below 2 spaces to the right to be consistent with other styles? 

```js
return css`
  display: inline-flex;
  // etc 
`
```

File: src/Form/Checkbox.stories.js [hunk 2]
```diff
@@ -1,52 +1,39 @@
...
 
 export default {
   title: 'Components|Forms/Checkbox',
   component: Checkbox,
 };
 
-export const Basic = () => (
-  <Checkbox id="checkbox" name="checkbox" label="I'm a checkbox" disabled={boolean('Disabled', false)} />
-);
+export const Basic = () => <Checkbox />;
```
Comment (erikshestopal) [inline src/Form/Checkbox.stories.js]: Can we add an example for a checkbox with the `isRadio` prop?

**Comment by vietnogigoodrx** [comment]
Should we make this wrap/responsive?

Screen Shot 2020-01-29 at 9 45 27 AM

**Comment by mathewmorris** [comment]
@erikshestopal Wanna give this another pass before merging?

File: src/Form/Checkbox.stories.js [hunk 3]
```diff
@@ -12,6 +12,8 @@ export const Basic = () => <Checkbox />;
 
 export const Label = () => <Checkbox label="I'm a checkbox" disabled={boolean('Disabled', false)} />;
 
+export const isRadio = () => <Checkbox isRadio />;
```
Comment (erikshestopal) [inline src/Form/Checkbox.stories.js]: I'm sorry, super nit: 
```suggestion
export const Radio = () => <Checkbox isRadio />;
```

Comment (mathewmorris) [reply src/Form/Checkbox.stories.js]: 😆 No worries!

**Pull request merged**